False positive in FieldableEntityInterface.stub

Copied from https://www.drupal.org/project/upgrade_status/issues/3295988.

There appears to be a false-positive being flagged in one of the stub classes.

``` ------ ----------------------------------------------------------------------- 
  Line   vendor/mglaman/phpstan-drupal/src/Drupal/../../stubs/Drupal/Core/Enti  
         ty/FieldableEntityInterface.stub                                       
 ------ ----------------------------------------------------------------------- 
  36     Method Drupal\Core\Entity\FieldableEntityInterface::getFields()        
         return type with generic interface                                     
         Drupal\Core\Field\FieldItemListInterface does not specify its types:   
         T                                                                      
  41     Method                                                                 
         Drupal\Core\Entity\FieldableEntityInterface::getTranslatableFields()   
         return type with generic interface                                     
         Drupal\Core\Field\FieldItemListInterface does not specify its types:   
         T                                                                      
 ------ ----------------------------------------------------------------------- 
 [ERROR] Found 2 errors     ```

--- a/stubs/Drupal/Core/Entity/FieldableEntityInterface.stub
+++ b/stubs/Drupal/Core/Entity/FieldableEntityInterface.stub
@@ -31,12 +31,12 @@ interface FieldableEntityInterface extends EntityInterface {
     public function toArray(): array;
 
     /**
-     * @return array<string, \Drupal\Core\Field\FieldItemListInterface>
+     * @return array<string, \Drupal\Core\Field\FieldItemListInterface<\Drupal\Core\Field\FieldItemInterface>>
      */
     public function getFields(bool $include_computed = TRUE): array;
 
     /**
-     * @return array<string, \Drupal\Core\Field\FieldItemListInterface>
+     * @return array<string, \Drupal\Core\Field\FieldItemListInterface<\Drupal\Core\Field\FieldItemInterface>>
      */
     public function getTranslatableFields(bool $include_computed = TRUE): array;
 }

